contracts/socket/SocketSrc.sol

```diff
@@ -62,6 +62,14 @@ abstract contract SocketSrc is SocketBase {
         );
 
         plugConfig.capacitor__.addPackedMessage(packedMessage);
+
+        _deductFees(
```

Comment:
I would call this something like `sendFees()`  to indicate that fees are being send to other contracts.

Reply:
Done in https://github.com/SocketDotTech/socket-DL/pull/234